django/applications/catmaid/static/js/WindowMaker.js

```diff
@@ -130,6 +130,13 @@ var WindowMaker = new function()
    * configuration.
    */
   var createWidget = function(instance) {
+    try {
+      CATMAID.throwOnInsufficientWebGlContexts(instance.MIN_WEBGL_CONTEXTS);
+    } catch (e) {
+      instance.unregisterInstance();
```

Comment:
Not all widgets register instances and be opened multiple times. This is unlikely a problem here, because the WebGL widgets are using `registerInstance()` though. Anyhow, I think it would be more robust to call the widget's `destroy()` method, because it should know best. And this should also only happen it is an actual function, i.e. `if (CATMAID.tools.isFn(instance.destroy)) { instance.destroy(); }`.